src/main.tsx

```diff
@@ -1,11 +1,74 @@
+import { ThemeProvider, createTheme } from "@mui/material";
 import React from "react";
 import ReactDOM from "react-dom";
 import { App } from "~/components";
 import "~/styles/index.css";
 
+const textCollor = "#FFFFFF";
+
+const theme = createTheme({
+  typography: {
+    h3: {
+      fontSize: "32px",
+      fontFamily: "Comfortaa",
```

Comment:
You can set the fontFamily property at the same level with h3, to set the font for all the MUI components, and remove the fontFamily prop from button and input overrides afterwards

Reply:
Done, worked fine